Client/src/modules/core/components/rent-a-yard/SoccerField/RentYardPage.jsx

```
@@ -2,63 +1,117 @@
-            { id: 'field-7-1', name: 'Sân 7 - Số 1', img: '/sanbong1.png' },
-            { id: 'field-7-2', name: 'Sân 7 - Số 2', img: '/sanbong2.png' },
-        ]
-        ,
-        '11': [
-            { id: 'field-11-1', name: 'Sân 11 - Số 1', img: '/san11.png' },
-        ]
+    const fetchFields = async (fieldType) => {
+        try {
+            setLoading(true);
+            const response = await axios.get(`http://localhost:8080/v1/fields/type/${fieldType}`);
+            if (response && response.data && response.data.length > 0) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)